### DISCUSSION TITLE
How much approx time github support time take to reply

### DISCUSSION THREAD

**Discussion by sohamdeogaonkar** [community/community#50477] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi.

GitHub suspended my account without any notice/email about suspending the account.
I don't know reason for suspension.

I raised a complaint against it.
But GitHub support team is not responding.

I raised a complain 3-4 days ago.

Still I don't have response from the team.

What are other options I could try?

**Comment by Anmol-Baranwal** [comment]
If your GitHub account has been suspended without notice or explanation, and you have not received a response from the GitHub support team, there are several steps you can take to try to resolve the issue:

1. Check your email spam folder: Sometimes emails from GitHub can end up in your spam folder, so it's worth checking there to see if you missed any communication.

2. Follow up with the support team: If you haven't received a response from the support team after a few days, you can follow up with another email or ticket to inquire about the status of your account.

3. Contact GitHub on social media: You can also try reaching out to GitHub on social media, such as Twitter or Facebook, to ask for assistance with your account.

Note:
The amount of time it takes for the GitHub support team to respond can vary depending on the volume of requests they are receiving and the complexity of the issue. In general, GitHub aims to respond to support requests within a few days, but it may take longer if there are a high number of requests.

**Reply by sohamdeogaonkar** [reply]
Its about 8-9 days completed .
Still they don't have replied.

I replied 2-3 times between this for any update.
but there is no update from their side

**Reply by Anmol-Baranwal** [reply]
I find it painful to hear this. 

@queenofcorgis, could you please assist him? You are the most suitable person for the task.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - thanks for the tag. All I can do is confirm your ticket is in the right place and response time will vary based on the number of tickets in front of you and our team's resources. Support is the only place that can assist with this, the Community is unable to help with account-related matters like this one so I'll be locking the post. Your patience is appreciated.
